src/evaluations/evaluator.py

```diff
@@ -224,14 +231,17 @@ def __init__(self, evaluation_config, sketch_estimator_config_list, run_name,
     self.scenario_random_states = scenario_random_states
 
   def __call__(self):
-    self.evaluate_all()
+    if self.parallel_cores > 1:
```

Comment:
Why keep evaluate_all()?

Reply:
This is a good point. I hadn't even thought about it, but I suppose the evaluate_all_parallel method works nearly exactly the same way that evaluate_all works. I will change this to only use evaluate_all_parallel.

Reply:
Resolved in 27cfae7